src/main/java/lotto/domain/LottoGroup.java

```diff
@@ -0,0 +1,60 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class LottoGroup {
+    private final List<Lotto> lottos;
+
+    public LottoGroup(Money money) {
+        this(buyLotto(money));
+    }
+
+    public LottoGroup(List<Lotto> lottos) {
+        this.lottos = lottos;
+    }
+    private static List<Lotto> buyLotto(Money money) {
+        int cnt = money.getBuyableCount();
+
+        List<Lotto> lottoArray = new ArrayList<>();
+
+        for(int i = 0; i < cnt; i++) {
+            lottoArray.add(Lotto.of(LottoMachine.createLottoNumbers()));
+        }
+
+        return lottoArray;
+    }
+
+    public List<Lotto> getLottoNumbers() {
+        return lottos;
+    }
+
+    public LottoResult match(Lotto winLotto) {
+        return new LottoResult(calculate(winLotto));
+    }
+
+    private  Map<LottoRank, Integer> calculate(Lotto winLotto) {
```

Comment:
```suggestion
    private  LottoResult calculate(Lotto winLotto) {
```
이미 Map을 포장한 객체가 있는데 굳이 Map으로 반환한 필요는 없지 않을까?